tests/lib/reset.sh

```diff
@@ -61,6 +61,12 @@ reset_classic() {
         exit 1
     fi
 
+    case "$SPREAD_SYSTEM" in
```

Comment:
could you please add a short comment to explain that line?

Reply:
Added now.